source/server/infrastructure/router.coffee

```diff
@@ -31,24 +32,38 @@ class Space.eventSourcing.Router extends Space.messaging.Controller
     if not @initializingCommand?
       throw new Error Router.ERRORS.missingInitializingCommand
     @routeCommands ?= []
+    @routeEvents ?= []
     super
 
   onDependenciesReady: ->
     super
+    @_setupInitializingCommand()
+    @_routeCommandToAggregate(commandType) for commandType in @routeCommands
+    @_routeEventToAggregate(eventType) for eventType in @routeEvents
```

Comment:
Honestly, im thinking about removing `Space.eventSourcing.Process` – it brings no real value to separate these because we are doing "Sagas" or "ProcessManagers" differently from other projects: they are also just event-sourced aggregates.